docs/assets/templates/locationID_template.json

```diff
@@ -0,0 +1,114 @@
+{
+	"description": "Hierarchical Encounter.locationID list in Wildbook",
+	"source": "http://en.wikipedia.org/wiki/List_of_seas",
```

Comment:
not gonna stop the PR for this, but we gotta remove "source" or at least make it accurate. This is not a List of seas from wikipedia 😆

Reply:
Fixed!